omnidotdev/zenpetal
# Fix latent Panda token collisions

## Description

##### Task link: https://linear.app/omnidev/issue/OMNI-39/fix-panda-dependency-040-breaking-styles

- Panda CSS 0.42 introduced a fix (more unique class names) to prevent token generation collisions. Without Sigil's peer deps being updated, this prevented styles generating from the Panda class names, causing mismatched styles or even styles not generating at all.

## Test Steps

- Verify the build from this PR properly generates *all* token styles. This can be tested by building and linking Sigil into a project and making sure styles are properly reflected for all Panda CSS classes (or corresponding JSX props)
